contents/uix/uix.md

```diff
@@ -45,3 +47,65 @@ Just like Software Development, UX Design is an iterative process. It involves m
+  * [User Flows](https://cdn-images-1.medium.com/max/1600/0*aO_6rai_fTnSe5rh.) - Depiction of how users navigate through the product for each scenario
+  
+### Prototyping
+Prototyping stage is where you:
+  * [Test](https://usabilitygeek.com/usability-testing-prototypes/) whether the product flow you came up with is smooth
+  * Create [prototypes](https://www.mockplus.com/blog/post/wireframe-mockup-prototype-selection-of-prototyping-tools), or Minimum Viable Product (MVP), for further testing and refinement
+  * Get a clear sense of how users would interact with the final product
+  
+#### Deliverables:
+  * [Wireframes](https://wireframesketcher.com/samples/YouTube.png) - Depicts a skeleton version of an application, meant to develop a unified vision of content structure instead of focusing on asthetics; may or may not be clickable
+  * [UI elements](https://thumbs.dreamstime.com/z/flat-ui-design-elements-set-icons-buttons-progress-bars-vector-illustration-light-colors-33417705.jpg) - Basic visual components, which can start out basic and increase in quantity/quality with the fidelity of MVP
+  * [Mockups](https://i0.wp.com/brandhorse.com/wp-content/uploads/2016/01/EZ-Frabic-Mobile-App-UX-UI-Design.jpg?resize=960%2C750) - Built upon an agreed version of wireframe with higher fidelity and asthetics
+  
+### User Testing
+User Testing stage is where you:
+  * Should have a clear sense of the user goals at this stage
+  * Use a low or high fidelity prototype to [test with users](http://www.uxbooth.com/articles/usability-testing-dont-guess-test/), to determine the effectiveness of product
+  * Be prepared to [re-prototype](https://www.interaction-design.org/literature/article/design-iteration-brings-powerful-results-so-do-it-again-designer) again after gathering user feedback
+  
+#### Deliverables (before testing):
+  * [Usability Test Plan](https://www.smileycat.com/wp-content/uploads/2016/05/usability-test-plan-dashboard.png) - Summary of background, goals, and test methodology
+  * [User Testing Script](http://www.ariadne.ac.uk/images/issue62-loureiroKoechlin/CeciliaLoureiroKoechlin-03-small.gif) - What to say as the facilitator, encouraging users to think out loud during the test
+  
+#### Deliverables (after testing):
+  * [Usability reports](https://image.slidesharecdn.com/january2012nhupa-120206210831-phpapp02/95/delivering-results-how-do-you-report-user-research-findings-18-728.jpg?cb=1374224669) - Thorough introduction to what aspects of the product was or was not effective and how to improve
+  * [Journey maps](https://cdn-images-1.medium.com/max/1600/1*jAfZXNWAx50fWEvoF-eWLw.png) - More detailed user flow based on how users are observed to use the product in actual testing conditions
+  * [Refined prototype](https://cdn-images-1.medium.com/max/1600/1*8qPOD1DG19Kj2sfJkTk3Eg.png) - A more refined prototype that is closer to meeting user expectations, each iteration should always be tested before moving to a higher fidelity
```

Comment:
Could use upper case for both words in the deliverable to standardize with the rest of the document.

eg: Refined Prototype, Usability Reports